.github/workflow-template/jobs/e2e-risedev.yml

```diff
@@ -102,6 +102,14 @@ jobs:
           sqllogictest -p 4566 './e2e_test/v2/ddl/**/*.slt'
           sqllogictest -p 4566 './e2e_test/v2/batch/**/*.slt'
 
+      - name: e2e test w/ Rust frontend ci-3node, source with kafka
```

Comment:
Can we put this job in the `source` workflow?

Reply:
Do you mean .github/workflow-template/jobs/e2e-source.yml? I'd prefer to call it e2e-regress, and it only runs the regression tests rather than the source tests. And it uses the java frontend. I'm considering completely removing it and we will have no ci workflow relying on java frontend.

Reply:
Get it.